### DISCUSSION TITLE
Is the new Copilot Metrics API only for Orgnaizations with "Copilot Business"?

### DISCUSSION THREAD

**Discussion by habibrahman29** [community/community#133846] [Actions] [answered]
### Select Topic Area

Question

### Body

So I received an email telling me:

As an administrator of an enterprise or organization using GitHub Copilot, you have been granted access to the GitHub Copilot Metrics API Open Beta.

I've got an Organization that is "upgraded" thanks to GitHub Education. Many of the repos in this Organization are assignments, which is effectively a number of students with (free) GitHub Copilot access (thanks to the GH Student Developer Pack).

The Organization itself isn't directly signed up to Copilot Business (no point, since all members are students/teachers and they anyway have free access --- albeit via their own personal GitHub accounts).

After running the API, I just get back an empty array. I'm assuming it's because it's not a Copilot Business organization?

However, I am confused as to why I received this email if I am not an admin of any Organization with Copilot Business.

Would appreciate some clarification: Is this API only for accessing Copilot metrics for Organizations that are effectively paying $19 per user in order to be Copilot Business? Or is it for any Organization that has users that might be using Copilot (albeit available to them either because they are students/teachers, or because they are paying for it themselves)?

**Comment by anzilparviz29** [comment, marked as answer]
It seems like you're encountering an issue because the GitHub Copilot metrics API is tailored for organizations that are subscribed to Copilot Business, which involves paying the $19 per user fee. Since your organization consists of students and teachers with free access through the GitHub Student Developer Pack, it's likely that these free accounts don't fall under the same category as paying Copilot Business users.

The API is designed to provide metrics specifically for organizations that are paying for Copilot Business. If your organization isn't classified as such, it makes sense that the API returns an empty array. The confusion might arise from receiving the email because your organization uses GitHub Education, and GitHub might be sending these communications to all organizations with any form of Copilot access.

For more detailed insights on this and how it affects educational organizations, you might find relevant discussions and articles under the business category.

**Comment by Hassan1712Ali** [comment]
My api
